src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -389,6 +404,13 @@ uct_dc_mlx5_iface_dci_release(uct_dc_mlx5_iface_t *iface, uint8_t dci_index)
     uint8_t pool_index           = uct_dc_mlx5_iface_dci_pool_index(iface,
                                                                     dci_index);
     uct_dc_mlx5_dci_pool_t *pool = &iface->tx.dci_pool[pool_index];
+    uct_dc_mlx5_ep_t *ep         = uct_dc_mlx5_ep_from_dci(iface, dci_index);
+
+    ucs_debug("iface %p: release dci %d with ep %p", iface, dci_index, ep);
+    if ((ep != NULL) && (ep->flags & (UCT_DC_MLX5_EP_FLAG_FLUSH_CANCEL |
+                                      UCT_DC_MLX5_EP_FLAG_INVALID))) {
```

Comment:
why added reset? it should already happen from error handler

Reply:
yes, need to add NOP like we do in FLUSH_CANCEL